handbook/engineering/security/index.md

```diff
@@ -4,9 +4,51 @@ We think that security is an enabler for the business. Sourcegraph is committed
 
 ## Goals
 
-Customers know they can confidently and securely use Sourcegraph with their private code.
+### Long term
 
-[This (internal) document](https://docs.google.com/document/d/1HhYfnkC17yCPqqw91H1QAF7q8toOOiNtoOOw06hqrNQ/) details our plan.
+Every organization, is confident they can trust Sourcegraph with the security of their Source code.
+
+### Medium term
+
+The goals below lie on the path to our long-term goals, and represent the next 3-6 months of work.  These goals will be delivered serially, but may be worked on in parallel.
+
+**We share our private [infrastructure repository](http://github.com/sourcegraph/infrastructure) on Sourcegraph cloud**
+
+  - Problem and rationale: While Sourcegraph cloud is in an good, and improvine security state, we currently do not have the right level of visibility. By starting with a private repository that contains no secrets, we can build and deploy the visibility technologies and processes that allow us to build a holistic view of the environment.  This serves as a proxy for small company repository use.
```

Comment:
```suggestion
  - Problem and rationale: While Sourcegraph cloud is in a good state in regards to security, we currently do not have the right level of visibility. We need more visibility into our threat surface before we feel confident putting private code in our cloud service. We can use our own comfort level as a proxy for how other small companies would feel when asked to put private code on cloud. We want to be able to say that we have good visibility into our threat surface, and we want to be confident that security issues will be detected and fixed before they become problematic.
```

Reworded this to hopefully make it flow better. Fundamentally, we want to get cloud into a state where we're confident in putting our own private code out there, and where we're confident that we'll know about vulnerabilities very quickly.